HW_1/Task3.py

```diff
@@ -0,0 +1,20 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. 
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+# *Пример:*
+# 385916 -> yes
+# 123456 -> no
+
+ticketNumber = int(input('Введите шестизначный номер билета :'))
+
+n1=ticketNumber//1000
+n2=ticketNumber%1000
+a = n1 // 10
+b = n2 // 10
+sumFirst = (n1 % 10)+(n1 // 100)+(a % 10)
+sumSecond = (n2 % 10)+(n2 // 100)+(b % 10)
```

Comment:
Никакие скобки тут не нужны.
```suggestion
sumFirst = n1 % 10 + n1 // 100 + a % 10
sumSecond = n2 % 10 + n2 // 100 + b % 10
```